scripts/cleanup_old_vms.py

```diff
@@ -29,9 +30,16 @@ def parse_cmd_line():
         '(varies by provider, default 24)')
     parser.add_argument('--provider', dest='providers', action='append', default=None,
         help='Provider(s) to inspect, can be used multiple times', metavar='PROVIDER')
-    parser.add_argument('text_to_match', nargs='*', default=['^test_', '^jenkins', '^i-'],
+    parser.add_argument('text_to_match', nargs='*', default=['^test', '^jenkins', '^i-'],
```

Comment:
https://media.tenor.co/images/24ac280565087507c57535c70bfae4d0/raw

Assuming you mean removing the '_' from test in the list ... Don't we have a mix of 'test-' and 'test_' now? Perhaps it'll be less dangerous if we specify '^test_' and '^test-' in the list?